CHANGELOG.md

```diff
@@ -1,2128 +1,13 @@
-  unnecessary code.
-- Updated copy on `about-us` page
-- Added copying of `.env_SAMPLE` to `.env` part of `setup.sh`.
-- Moved console output messages to the end of the `setup.sh` `init` method.
-- Organized `.env_SAMPLE` and made `.env` executable on its own.
-- Added `HTTP_HOST`, `HTTP_PORT`, `SELENIUM_URL`, `SAUCE_USERNAME`,
-  `SAUCE_ACCESS_KEY`, `SAUCE_SELENIUM_URL`, and `VIRTUAL_ENV`
-  constants to `.env_SAMPLE`.
-- Moved aggregate `gulp lint` task to bottom of file to avoid duplicate
-  lint task entries in `gulp --tasks`.
-- Renamed `gulp lint:src` to `gulp lint:scripts` to future-proof type of linting.
-- Renamed `gulp test:macro` to `gulp test:unit:macro`.
-- Renamed `gulp test:processor` to `gulp test:unit:processor`.
-- Renamed `gulp test:browser` to `gulp test:acceptance:browser`.
-- Edited `INSTALL.md` to accommodate changes in `.env_SAMPLE`.
-- Edited Protractor configuration to include browser groups,
-  which by default only run the essentials locally, but the full suite
-  (including legacy browsers) on Sauce Labs when Sauce credentials are present.
-- Updated test instructions to use the gulp test subtasks.
-- Updated Travis CI settings to use `setup.sh`.
-- Updated files to use `breakpoints-config.js`.
-- Made `/the-bureau/bureau-structure/role-macro.html` private.
-- Updated `gulp clean` to leave the `dist` directory and remove the inner
-  contents
-- Use `HTTP_PORT` environment variable for port in `gulp watch`, if available.
-- Removed "optional" text from privacy complaint form
-  and added `*` to designate required fields.
-- Updated Deputy Director information to Meredith Fuchs.
-- Updated `/about-rich-cordray/` URL to `/about-director/`.
-- Updated `/about-meredith-fuchs/` URL to `/about-deputy-director/`.
-- Normalized director and deputy director photos to be format `NAME-WxH.jpg`.
-- Changed name of `shallow-extend` utility to 'assign'.
-- Superscripts `st` in `21st` on About Us page.
-- Updated `BreakpointHandler.js` to support usage of `breakpoints-config.js`.
-
-### Removed
-- Removed styles from codebase that have already been migrated
-  to cf-typography.
-- Removed duplicate Privacy Policy
-- Removed processor tests due to them being outdated.
-- Removed failing bureau tests to be debugged later
-
-### Fixed
-- Fixed borders on sub-footers across the website
-- Fixed 'Return to top' button width on footer
-- Fixed default gulp task
-- Fixed icon links to match CFPB Design Manual
-- Fixed gulp copy task that was missing copying PDFs in subdirectories.
-- Fixed issues with active filter logic.
-- Fixed testing issue with single pages reloading for every test
-- Fixed testing timeouts the first fix didn't correct by updating timeout time
-
-
-## 3.0.0-2.2.0 - 2015-08-18
-
-### Added
-- Transitioned Capital Framework dependency to v1.0.0 in bower.json.
-- Added gulp and the required npm plugins
-- Added gulp config file to lay out configs for each task
-- Added gulp task
```

Comment:
Sorry, put this on the wrong line. It should be referring to the shell command line.